### ISSUE TITLE
Add PHP 7.3 support

### ISSUE THREAD

**Issue by rybakit** [tarantool/tarantool-php#139]
Ref: https://blog.remirepo.net/post/2018/07/02/PHP-extensions-status-with-upcoming-PHP-7.3

**Comment by Totktonada** [comment]
A user observed the build fail:

```
/var/tmp/tarantool/src/tarantool.c: In function ‘zim_Tarantool___construct’:
/var/tmp/tarantool/src/tarantool.c:1153:20: error: lvalue required as left operand of assignment
   GC_REFCOUNT(&le) = 1;
                    ^
Makefile:193: recipe for target 'src/tarantool.lo' failed
```

It seems that we should use `GC_SET_REFCOUNT(&le, 1);` on PHP 7.3, see https://github.com/couchbase/php-couchbase/pull/22/files#diff-0681ab7d7064f2e8b779813c9c114005R288

**Comment by bigbes** [comment]
https://cismon.net/2018/02/06/Changes-of-Zend-API-in-PHP-7.3/

**Comment by SonSergei** [comment]
See https://github.com/tarantool/tarantool-php/pull/148

**Comment by Totktonada** [comment]
Fixed in `php7-v2` branch in [0.3.0-24-g3d0962b](3d0962b47264768a73558c27367c8f299190b59f). It'll be switched to `master` in the scope of #137.
